package.json

```diff
@@ -15,8 +15,8 @@
   "types": "index.d.js",
   "scripts": {
     "lint": "xo",
-    "test:unit": "mocha test/unit/**",
-    "test": "xo && nyc mocha test/unit/**",
+    "test:unit": "jest",
+    "test": "jest --coverage",
```

Comment:
nyc is the cli for Istanbul coverage tool and that is what jest uses with the `--coverage` flag. I decided to split the linting away from tests so that local runs are faster and CI can run the scripts separately.